lab-nicu/test/exercise-router-test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+require('dotenv').config({ path: `${process.cwd()}/.test.env` });
+require('./lib/mock-aws.js');
+
+const expect = require('expect');
+const superagent = require('superagent');
+
+const server = require('../lib/server.js');
+const clearDB = require('./lib/clear-db.js');
+const mockUser = require('./lib/mock-user.js');
+
+const API_URL = `${process.env.API_URL}:${process.env.PORT}`;
+
+describe('testing exercise router', () => {
+  before(server.start);
+  after(server.stop);
+  afterEach(clearDB);
+
+  describe('testing POST /api/exercise', () => {
+    it('should respond with an exercise and 200', () => {
```

Comment:
I can't get this test to pass. It might be a timeout issue but I just get a 500 error.